third_party/crashpad/tools/BUILD.gn

```diff
@@ -29,9 +29,16 @@ source_set("tool_support") {
   }
 }
 
-# TODO(b/251521595): resolve GN error and enable this target to be built for
-# android.
-if (!crashpad_is_ios && !crashpad_is_android) {
+if (!crashpad_is_ios) {
+  if (crashpad_is_in_native_target_build) {
+    config("crashpad_database_util_config") {
+      # This is to undo the "main=StarboardMain" define added for all targets
+      # when final_executable_type == "shared_library", in
+      # starboard/build/config/BUILD.gn, which itself is admittedly a hack.
+      defines = [ "StarboardMain=main" ]
```

Comment:
We have `crashpad_handler_native_target_config` which does the same thing—do we want to reuse it? (maybe it can go in a shared location?)

Reply:
Sounds good to me to share it. Let me know what you think of the location - an argument could probably also be made for `third_party/crashpad/build/shared/BUILD.gn`.

Reply:
I think putting it in the existing `third_party/crashpad/build/BUILD.gn` works, it already seems to have that purpose

Reply:
Works for me - done.